app/models/provider_relationship_permissions.rb

```diff
@@ -2,7 +2,18 @@ class ProviderRelationshipPermissions < ApplicationRecord
   belongs_to :ratifying_provider, class_name: 'Provider'
   belongs_to :training_provider, class_name: 'Provider'
 
-  PERMISSIONS = %w[make_decisions view_safeguarding_information].freeze
+  PERMISSIONS = %i[make_decisions view_safeguarding_information].freeze
+
+  validate :at_least_one_active_permission_in_pair, if: -> { setup_at.present? }
+
+  def self.permissions_fields
```

Comment:
Do we use this anywhere?

Reply:
@duncanjbrown Regarding validation - _I think_ the heroku provider relationships all have `setup_at: nil`, the initial state of a provider-to-provider relationship, this means the validation won't add any errors. Validation works for me locally.
We should never have this situation once the redirect to the permissions setup screens are back in place as this process sets that date.
I've removed the `permissions_fields` method.